### PR TITLE
feat(Copr): Improved installation instructions

### LINKED ISSUE #470: misleading instructions how to use the copr builds
### What happened? What is the problem?

Ohai,

when you have packit build a PR, the users get a link to the dashboard `https://dashboard.packit.dev/jobs/copr/:id` (e.g. https://dashboard.packit.dev/jobs/copr/1933132) that contains a message how to use the resulting builds, however, those are (possibly) misleading for users not very well versed in copr/dnf.

## `dnf copr enable {data.copr_owner}/{data.copr_project}` without `chroot`

Right now, the instructions say to call `sudo dnf copr enable packit/theforeman-foreman_maintain-936`, which on EL8 will try to use the `epel-8-x86_64` chroot by default (and `epel-9-x86_64` on EL9), but Packit didn't build for those chroots (the linked build is for `rhel-8-x86_64`) and the user ends up with an error:
```
Error: It wasn't possible to enable this project.
Repository 'epel-8-x86_64' does not exist in project 'packit/theforeman-foreman_maintain-936'.
Available repositories: 'rhel-9-x86_64', 'rhel-8-x86_64'
```

Given Packit knows which chroot the build was for, it could add that to the `dnf copr` invocation, right?

## `dnf install -y <all packages>`

This might be a tad specific to @theforeman packages, but we do have quite a lot sub-packages that get built, but are not needed on a user system (they contain code and dependencies that are needed to build plugins etc).

The instructions on the copr job view tell the user (who might be unaware of our package split!) to `dnf install -y all-the-packages-that-were-built` which in the best case results in the user installing things they don't need and in the worst case results in dependency errors from `dnf` as we don't ship all packages required for *building* in our main repos.

It would be cool if the wording would point out that this might not what you need, or allow projects to override this messaging in their `packit.yml`

### What did you expect to happen?

_No response_

### Example URL(s)

https://dashboard.packit.dev/jobs/copr/1933132

### Steps to reproduce

```bash
1. tell users "take the packit build from <PR link>"
2. see them following the instructions literally
```

### Workaround

- [X] There is an existing workaround that can be used until this issue is fixed.

### Participation

- [ ] I am willing to submit a pull request for this issue. (Packit team is happy to help!)

### PR DESCRIPTION
**Pull request by Venefilyn**
This splits the packages up in easier chunks for installation.

Fixes #470

image

RELEASE NOTES BEGIN
Improve Copr installation instructions
RELEASE NOTES END

### CODE REVIEW

**Review by lachmanfrantisek — APPROVED** [review]
Nice, this is handy!

File: frontend/src/components/copr/CoprBuild.tsx [hunk 1]
```diff
@@ -79,32 +92,77 @@ export const CoprBuild = () => {
...
+                  <Title headingLevel="h3">Repository</Title>
+                  <List isPlain>
+                    <ListItem>
+                      <ClipboardCopy
+                        hoverTip="Copy"
+                        clickTip="Copied"
+                        variant="inline-compact"
+                        isCode
+                      >
+                        sudo dnf install -y dnf-plugins-core
```
Comment (nforro) [inline frontend/src/components/copr/CoprBuild.tsx:109]: Just a note, on F41+ the plugin is in `dnf5-plugins`, but it should be installed automatically.

**Review by lbarcziova — APPROVED** [review]
looks good!

do you know what might be causing the deploy-preview to fail? (briefly checked and it happened for past few PRs already)

File: frontend/src/components/copr/CoprBuild.tsx (hunk 1 above)
Comment (nforro) [reply frontend/src/components/copr/CoprBuild.tsx:109]: `dnf -y install 'dnf-command(copr)'` would be safer, though funnily enough `dnf5-plugins` doesn't provide it (but I'm sure they'll fix it eventually).

**Comment by Venefilyn** [comment]
@lbarcziova should be fixed in #488

File: frontend/src/components/copr/CoprBuild.tsx (hunk 1 above)
Comment (nforro) [reply frontend/src/components/copr/CoprBuild.tsx:109]: OTOH it seems to work everywhere (where there is `dnf`). @packit/the-packit-team WDYT?

```suggestion
                        sudo dnf install -y 'dnf*-command(copr)'
```

Comment (lbarcziova) [reply frontend/src/components/copr/CoprBuild.tsx:109]: I like the suggestion

Comment (Venefilyn) [reply frontend/src/components/copr/CoprBuild.tsx]: @nforro @lbarcziova added!

**Pull request merged**